mmdet/models/backbones/detectors_resnet.py

```diff
@@ -56,12 +57,8 @@ def __init__(self,
                 1,
                 stride=1,
                 bias=True)
-        self.init_weights()
-
-    def init_weights(self):
-        """Initialize the weights."""
-        if self.rfp_inplanes:
-            constant_init(self.rfp_conv, 0)
+            self.init_cfg = dict(
+                override=dict(type='Constant', val=0, name='rfp_conv'))
```

Comment:
in __init__ 
self.init_cfg = dict(type='Constant', val=0), override=dict(name='rfp_conv'))
if inherit `BaseModule`, there is no need to call init_weights(),